core/domain/exp_jobs_one_off_test.py

```diff
@@ -1693,16 +1692,6 @@ def test_for_validation_job(self):
         """Tests that the exploration validation job validates the content
         without skipping any tags.
         """
-
-        # This mock should only be used for
-        # ExplorationContentValidationJobForCKEditor.
-        # The job finds invalid strings in an exploration.
-        # If we do not use the mock, some of the strings will be converted
-        # to a valid format during initialization of subtitled html
-        # in state.
-        def mock_convert_to_ckeditor(html_data):
```

Comment:
Ditto

Reply:
Please do this for other parts where code was deleted.

Reply:
This mock function was not called and the test runs fine without this mock.

Reply:
@lilithxxx I think `mock_convert_to_ckeditor_context` was using this method earlier, can you please check why we have to delete?

Reply:
Yes I meant the `update_content` method was not using this mock